release-notes/major9/09.25.0/valtimo-frontend-libraries.md

```diff
@@ -63,4 +62,15 @@ No new deprecations.
 
 ## Known issues
 
-No new known issues.
+This version has the following known issues:
+
+* **Case search fields**
+  * Discovered in version 5.14.0 RC
+    * Some case search fields can be configured, but lead to errors when performing searches (i.e. boolean range).
+    * Date/time searches do not always function correctly because of time zone issues.
+    * After a 500 error occurs on a search action, the front-end does not recover, and the page has to be refreshed in
```

Comment:
Maybe explain in what situations the errors occur so developers can prevent that situation from happening. Is this only happening in relation to the range configurations?